### PR TITLE
docs: Add `wxt init` demo GIF

### PR DESCRIPTION
**Pull request by okineadev**
I've added a [VHS tape](https://github.com/charmbracelet/vhs) with autosync output files (`demo.gif` for now) using GitHub Actions to demonstrate project initialization with **WXT**

### CODE REVIEW

**Comment by okineadev** [comment]
Don't look at the errors

@aklinker1 Please review

**Comment by okineadev** [comment]
I fix it later

**Comment by okineadev** [comment]
It works!

**Comment by okineadev** [comment]
image

# We need to fix it

@aklinker1 , Do you know how to preinstall `wxt@latest` so that when running `pnpm dlx wxt@latest init .` it does not have to load it during recording?

File: docs/tapes/demo.tape [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Env PS1 "\e[0;32m●\e[0m "
+
+
+# ===== Preparation =====
+# Steps to prepare the recording environment, ensuring a clean and isolated setup.
+
+Hide
+# Create a temporary folder for demo
+Type 'vhs_sandbox="$(mktemp -d)"' Enter
+Type 'cd "$vhs_sandbox"' Enter
```
Comment (aklinker1) [inline docs/tapes/demo.tape:39]:
```suggestion
Type 'cd "$vhs_sandbox"' Enter
Type 'pnpm store add wxt@latest' Enter
```

This gets rid of the downloads!

**Review by aklinker1 — CHANGES_REQUESTED** [review]
Nice, this is super cool! Couple of changes for consistency, and I figured out how to do it without PNPM fetching the latest value - add it to the store inside the tape.

File: docs/public/demo.gif
[the remark this answers is not in the document]
Comment (aklinker1) [reply docs/public/demo.gif:1]: Resolved by https://github.com/wxt-dev/wxt/pull/974#discussion_r1752873565

File: .github/workflows/vhs.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4.0.0
+        with:
+          run_install: false
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4.0.3
+        with:
+          node-version: 20
+          cache: 'pnpm'
```
[the remark this answers is not in the document]
Comment (okineadev) [reply .github/workflows/vhs.yml:24]: No.

File: docs/tapes/demo.tape (hunk 1 above)
Comment (okineadev) [reply docs/tapes/demo.tape:39]: It doesn't work

**Comment by okineadev** [comment]
[jpg

**Comment by okineadev** [comment]
WOW IT WORKS!

File: docs/tapes/demo.tape (hunk 1 above)
Comment (aklinker1) [reply docs/tapes/demo.tape:39]: Hmm, it worked just fine for me when I generated the tape locally?

Comment (okineadev) [reply docs/tapes/demo.tape:39]: > Hmm, it worked just fine for me when I generated the tape locally?

Fixed in [`e99b7ec` (#974)](https://github.com/wxt-dev/wxt/pull/974/commits/e99b7ec84b2123433c65594ff0fc0593a3a8f58a).

File: .github/workflows/vhs.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (aklinker1) [reply .github/workflows/vhs.yml:24]: What do you mean "no"?

**Review by aklinker1 — APPROVED** [review]

**Comment by okineadev** [comment]
GitHub is:

**Pull request merged**